model/flow/header.go

```diff
@@ -72,6 +74,10 @@ func (h Header) Fingerprint() []byte {
 	return fingerprint.Fingerprint(h.Body())
 }
 
+var mutex_hdr sync.Mutex
+var previd_hdr Identifier
+var prev_hdr Header
```

Comment:
though I'm a big fan of snake_casing, camelCasing is what we have to follow for go codes.